### ISSUE TITLE
flutter doctor command on UI cannot find device and android_sdk

### ISSUE THREAD

**Issue by ferdavs** [flutter/flutter#10671]
## Steps to Reproduce

1. install flutter plugin to intellij or webstorm(versions are below)
2. create flutter project 
4. open  pubscpec.yaml file
3. while on pubscpec.yaml file try to run 'flutter doctor' command

IntelliJ IDEA 2017.1.4
Build #IU-171.4694.23, built on June 6, 2017

WebStorm 2017.1.4
Build #WS-171.4694.29, built on June 7, 2017

## flutter doctor UI command output

[✓] Flutter (on Linux, locale en_US.UTF-8, channel master)
    • Flutter at /home/f/App/dart/flutter
    • Framework revision 0774c5194b (2 hours ago), 2017-06-13 10:51:17 -0700
    • Engine revision 4f5d6fab11
    • Tools Dart version 1.24.0-dev.6.7

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.io/setup/#android-setup for detailed instructions).
      If Android SDK has been installed to a custom location, set $ANDROID_HOME to that location.

[✓] Android Studio (version 2.3)
    • Android Studio at /home/f/App/android-studio
    • Gradle version 3.2
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] WebStorm (version 2017.1)
    • Flutter plugin version 14.0

[✓] IntelliJ IDEA Ultimate Edition (version 2017.1)
    • Flutter plugin version 14.0
    • Dart plugin version 171.4694.29

[✓] Connected devices
    • None

## flutter doctor terminal output

[✓] Flutter (on Linux, locale en_US.UTF-8, channel master)
    • Flutter at /home/f/App/dart/flutter
    • Framework revision 0774c5194b (2 hours ago), 2017-06-13 10:51:17 -0700
    • Engine revision 4f5d6fab11
    • Tools Dart version 1.24.0-dev.6.7

[✓] Android toolchain - develop for Android devices (Android SDK 26.0.0)
    • Android SDK at /home/f/App/android-sdk-linux
    • Platform android-26, build-tools 26.0.0
    • ANDROID_HOME = /home/f/App/android-sdk-linux
    • Java binary at: /home/f/App/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] Android Studio (version 2.3)
    • Android Studio at /home/f/App/android-studio
    • Gradle version 3.2
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] WebStorm (version 2017.1)
    • Flutter plugin version 14.0

[✓] IntelliJ IDEA Ultimate Edition (version 2017.1)
    • Flutter plugin version 14.0
    • Dart plugin version 171.4694.29

[✓] Connected devices
    • GT N8000  • android-arm • Android 7.1.1 (API 25)

**Comment by eseidelGoogle** [comment]
Presumably in the terminal you're setting ANDROID_HOME but not in the environment from which the IDE is being launched?

**Comment by ferdavs** [comment]
Thank you
running webstorm from terminal solved.
I was running from unity icon created.

**Comment by FallenRiteMonk** [comment]
Running from terminal solved the problem also for me, but its kind of annoying to always having to open IDEA from the terminal.
Is there any possibility to setting this up some other way?

I have tried adding the android plugin in IDEA and setting the SDK path (Settings>Appearance & Behavior>System Settings>Android SDK>Android SDK location) and I also tried setting ANDROID_HOME as path variable (Settings>Appearance & Behavior>Path Variables).

Any solution or further ideas to try?
